### DISCUSSION TITLE
Volume Encryption?

### DISCUSSION THREAD

**Discussion by JustinGuese** [kube-hetzner/terraform-hcloud-kube-hetzner#287] [Ideas]
do we have an idea of how to implement volume encryption?
any ideas how we could solve it?

**Comment by JustinGuese** [comment]
Sry didn't see the discussion tab ;)

**Reply by mysticaltech** [reply]
Hello @JustinGuese, according to this, it's doable via Longhorn. I will look into enabling the complete set of helm values for Longhorn so it can be activated. I will keep you posted!

**Reply by mysticaltech** [reply]
https://longhorn.io/blog/longhorn-v1.2/

**Reply by JustinGuese** [reply]
would anyone be willing to implement it if we sponsor that feature? not too much budget though

**Reply by mysticaltech** [reply]
@JustinGuese I was planning to add a donate button to the project. I will speed this up! Donations are always appreciated but not a must. I will look more into actively adding encryption. Thanks for the offer!

**Reply by JustinGuese** [reply]
yeah i'm not sure if we can do it, but it's not too urgent, don't worry ;)

**Reply by mysticaltech** [reply]
I have explained how to do it here https://github.com/kube-hetzner/terraform-hcloud-kube-hetzner/issues/295#issuecomment-1235405678.

**Comment by mysticaltech** [comment]
This is now supported.
